### PR TITLE
Harshasi/unify pq files into one file rebased

### PR DESCRIPTION
**Pull request by harsha-simhadri**

### CODE REVIEW

**Review by rakri — COMMENTED** [review]
Looks ok to me code-wise. Will run some offline tests and update.

File: CMakeLists.txt [hunk 1]
```diff
@@ -42,7 +42,7 @@ if (MSVC)
 
     message(STATUS "Invoking nuget to download Boost, OpenMP and MKL dependencies...")
     configure_file(${PROJECT_SOURCE_DIR}/windows/packages.config.in ${DISKANN_MSVC_PACKAGES_CONFIG})
-    exec_program(${NUGET_EXE} ARGS install ${DISKANN_MSVC_PACKAGES_CONFIG} -ExcludeVersion -OutputDirectory ${DISKANN_MSVC_PACKAGES})
+    exec_program(${NUGET_EXE} ARGS install \"${DISKANN_MSVC_PACKAGES_CONFIG}\" -ExcludeVersion -OutputDirectory \"${DISKANN_MSVC_PACKAGES}\")
```
Comment (harsha-simhadri) [inline CMakeLists.txt:45]: What is the reason for this change?

Comment (rakri) [reply CMakeLists.txt:45]: if the build folder has spaces like "ANNS Project" it the nuget will treat it as separate args instead of one folder argument if we didn't have quotes

Comment (rakri) [reply CMakeLists.txt:45]: learned this the hard way :)

**Review by rakri — APPROVED** [review]
Looks good to me.
